res/skins/Deere/tool_bar.xml

```diff
@@ -103,6 +107,28 @@
             </Children>
           </WidgetGroup>
 
+          <WidgetGroup>
+            <ObjectName>BoothControls</ObjectName>
+            <Layout>horizontal</Layout>
+            <Connection>
+              <ConfigKey>[Master],booth_enabled</ConfigKey>
+              <BindProperty>visible</BindProperty>
+            </Connection>
+            <Children>
+              <Template src="skin:spacer_hx.xml">
+                <SetVariable name="width">15</SetVariable>
+              </Template>
+
+              <Template src="skin:knob_toolbar.xml">
+                <SetVariable name="TooltipId">master_gain</SetVariable>
+                <SetVariable name="group">[Master]</SetVariable>
+                <SetVariable name="control">booth_gain</SetVariable>
+                <SetVariable name="color">red</SetVariable>
+                <SetVariable name="label">Booth</SetVariable>
```

Comment:
Now we have Gain / Head Gain / Booth .. three naming schemas for similar features.

Reply:
Mmm... haven't checked it at all, but here is my opinion:
"Gain", without any other label, should be used where the context limits it to a specific control. I.e. track gain, sampler gain,  Master gain should tell that it is the Master.
"Head gain": if not using a hardware control (present in many controllers) the UI specific control for headphones should be labeled this way.
"Booth gain": AFAIK, the booth is the speakers/monitors that the DJ can have inside the cabin to listen to the audio. They are of importance when the cabin's position can't give the DJ enough detail. It can serve two purposes: to better equalize, or to better mix (getting a cleaner source of what's being played while also listening to the headphones). Some other times it might simply be for DJ amusement :P

Anyway, booth gain does not have much sense on Mixxx since we don't have an explicit Booth output and it should not be confused with headphones output. Booth is generally connected to the secondary output of a Mixer table, containing also the master output.

Reply:
I agree that "GAIN" without any context is not clear. I have changed it to "MASTER".

The booth output was introduced in #1279.